cpp/oneapi/dal/algo/kmeans_init/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -14,91 +14,67 @@
 * limitations under the License.
 *******************************************************************************/
 
-#include <include/algorithms/kmeans/kmeans_init_types.h>
-#include <src/algorithms/kmeans/oneapi/kmeans_init_dense_batch_kernel_ucapi.h>
-
 #include "oneapi/dal/algo/kmeans_init/backend/gpu/compute_kernel.hpp"
-#include "oneapi/dal/algo/kmeans_init/backend/to_daal_method.hpp"
+#include "oneapi/dal/algo/kmeans_init/backend/gpu/compute_kernels_impl.hpp"
 #include "oneapi/dal/backend/interop/common_dpc.hpp"
 #include "oneapi/dal/backend/interop/error_converter.hpp"
-#include "oneapi/dal/backend/interop/table_conversion.hpp"
 #include "oneapi/dal/detail/error_messages.hpp"
 #include "oneapi/dal/table/row_accessor.hpp"
 
 namespace oneapi::dal::kmeans_init::backend {
 
-using std::int64_t;
 using dal::backend::context_gpu;
 
-namespace daal_kmeans_init = daal::algorithms::kmeans::init;
 namespace interop = dal::backend::interop;
+namespace pr = dal::backend::primitives;
+
+template <typename Task>
+using result_t = compute_result<Task>;
```

Comment:
Looks like we will have only one task for foreseeable future, so the `Task` can be substituted with concrete type. The same is valid for the rest of the kernel.

Reply:
Fixed